### PR TITLE
Retry ssh commands

### PR DESCRIPTION
**Pull request by lasarojc**
Retry up to 5 times before failing the tasks.
Also add an install playbook, which aggregates multiple simpler playbooks.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -57,43 +57,49 @@ configgen:
...
 	cd ansible && \
-		ansible-playbook -i hosts -u root base.yaml -f $(ANSIBLE_FORKS) && \
-		ansible-playbook -i hosts -u root prometheus-node-exporter.yaml -f $(ANSIBLE_FORKS) && \
-		ansible-playbook -i hosts -u root init-testapp.yaml -f $(ANSIBLE_FORKS) && \
-		ansible-playbook -i hosts -u root update-testapp.yaml -f $(ANSIBLE_FORKS) -e "version_tag=$(VERSION_TAG)" -e "go_modules_token=$(GO_MODULES_TOKEN)"
+		ANSIBLE_SSH_RETRIES=5 ansible-playbook -i hosts -u root install.yaml -f $(ANSIBLE_FORKS) -e "version_tag=$(VERSION_TAG)" -e "go_modules_token=$(GO_MODULES_TOKEN)"
 ifneq ($(VERSION2_WEIGHT), 0)
 	cd ansible && \
-		ansible-playbook -i hosts --limit validators2 -u root update-testapp.yaml -f $(ANSIBLE_FORKS) -e "version_tag=$(VERSION_TAG2)" -e "go_modules_token=$(GO_MODULES_TOKEN)"
+		ANSIBLE_SSH_RETRIES=5 ansible-playbook -i hosts --limit validators2 -u root update-testapp.yaml -f $(ANSIBLE_FORKS) -e "version_tag=$(VERSION2_TAG)" -e "go_modules_token=$(GO_MODULES_TOKEN)"
```
Comment (sergio-mena) [inline Makefile:63]: Can you make the value (i.e. "5") a makefile variable? (I would like to leverage this with particular ssh agent setup, and I suspect I'll have to increase it)

**Review by sergio-mena — APPROVED** [review]
Thanks for this!

**Comment by sergio-mena** [comment]
@lasarojc , thanks for the final change!
When I read it, I had a reasonable doubt if this would work (as I was not 100% sure whether Makefile variables are propagated as env variables by default). So I did the following test
```
sergio@informal-sergio:mydir % cat Makefile 
MYVAR=1

do1:
	bash -c 'echo value: $$MYVAR'
do2:
	echo value $(MYVAR)
do3:
	MYVAR=$(MYVAR) bash -c 'echo value: $$MYVAR'
sergio@informal-sergio:mydir % make do1
bash -c 'echo value: $MYVAR'
value:
sergio@informal-sergio:mydir % make do2
echo value 1
value 1
sergio@informal-sergio:mydir % make do3
MYVAR=1 bash -c 'echo value: $MYVAR'
value: 1
sergio@informal-sergio:mydir %
```

So, it seems that Makefile variables are not propagated as env variables.

**Comment by lasarojc** [comment]
I see.
I should have done a similar experiment. So in the final version, we are missing assignments from the Makefile variables to env variables. You said you were going to use these changes for something else. Will you include the fix for this omission, or do you want me to create another PR?

**Comment by lasarojc** [comment]
```make
export MYVAR=1

do1:
	bash -c 'echo value: $$MYVAR'
do2:
	echo value $(MYVAR)
do3:
	MYVAR=$(MYVAR) bash -c 'echo value: $$MYVAR'
```

```
{8:07}~ ➭ make do1
bash -c 'echo value: $MYVAR'
value: 1
{8:07}~ ➭ make do2
echo value 1
value 1
{8:08}~ ➭ make do3
MYVAR=1 bash -c 'echo value: $MYVAR'
value: 1
```

**Comment by sergio-mena** [comment]
>You said you were going to use these changes for something else

I was just thinking of my whimsical ssh-agent :-)

I'm not opening a PR in this repo shortly. Next time, it will probably be for `v0.38.x` QA, hopefully within this month... maybe we can wait until then? If that looks good for you, no need to add an issue now, as my ssh-agent will make sure I don't forget :-)

**Pull request merged**